starlette/responses.py

```diff
@@ -207,11 +207,13 @@ def __init__(
         background: BackgroundTask = None,
         filename: str = None,
         stat_result: os.stat_result = None,
+        send_header_only: bool = False,
     ) -> None:
         assert aiofiles is not None, "'aiofiles' must be installed to use FileResponse"
         self.path = path
         self.status_code = 200
         self.filename = filename
+        self.send_header_only = send_header_only
```

Comment:
Then we can do `self.send_header_only = method is not None and method.upper() == "HEAD"`